### DISCUSSION TITLE
Feedback loop from taker to maker

### DISCUSSION THREAD

**Discussion by da-kami** [get10101/itchysats#1672] [General]
For features like #1658 it would be great to have a feedback loop from the taker to the maker to see if a taker (unexpectedly) runs into a scenarios where the taker thinks an offer is outdated. 
At the moment the maker does not receive any feedback from the taker. 

This feature is not super critical, but would be nice to have to learn more about the behavior of the taker daemon.

**Comment by thomaseizinger** [comment]
Converted this to a discussion because the idea is to immature to implement.

**Comment by klochowicz** [comment]
The easiest way of implementing this would be introducing a new message from the taker, something like: `RequestCurrentOffer`, optionally with some reason (which could be an enum or even a string).
This does not take care of any other places where we'd like to have feedback, but I'm not sure if there are any other ones?

**Comment by bonomat** [comment]
The discussion title sounds general but the description asks for a feedback loop to the maker if an order is outdated. 
@da-kami : can we close this discussion ? If not, can you rephrase the description to make it more general please?

**Reply by da-kami** [reply]
Feel free to close it. We can open a separate discussion or issue when needed.

**Comment by holzeis** [comment]
Looking over our protocol implementations, I found that we generally do not handle scenarios aside from the happy path very well. I think we generally miss a protocol step to report back if anything went wrong. Currently we would only end up in a timeout.

e.g. working on the  #2891 for instance gives me a hard time to fetch the state of the failed collaborative settlement proposal.
